common/lang.ts

```diff
@@ -0,0 +1 @@
+export const LanguageCodes: Array<string> = ['ar', 'zh', 'en', 'fr', 'ru', 'es']
```

Comment:
This exists currently in core/common/lang.ts

Reply:
core/common should not exists. we will remove that

Reply:
plus, that's not an array of available languages. will move LanguageCodes it under core